pkg/volume/aws_ebs/aws_ebs.go

```diff
@@ -291,6 +290,10 @@ func (b *awsElasticBlockStoreBuilder) SetUpAt(dir string) error {
 		return err
 	}
 
+	if !b.readOnly {
```

Comment:
My point was that the post-condition is that ownership is set.  If fsGroup is 0, we need to ensure that everything is gid=0.  That *should* be the default today, but I don't know if it is safe to assume it always will be.